src/restricted_app/restricted_host_interface.c

```diff
@@ -115,7 +115,6 @@ void restricted_host_interface(engine_ctx_t *ctx,
   /* Device authentication is complete, reset the flow as the core will now need
    * to render the main menu */
   if (DEVICE_AUTHENTICATED == get_auth_state()) {
-    delay_scr_init(ui_text_check_cysync_app, DELAY_TIME);
```

Comment:
is this intentional? link any request for change to this.

Reply:
Yes, once we complete the authentication process, this screen is displayed, which seems unnecessary.

Reply:
yes, that makes sense. does not seem as an issue.

just needed to double check.